plugin/src/makeWorklet.ts

```diff
@@ -244,6 +257,16 @@ export function makeWorklet(
   return newFun;
 }
 
+function shouldInjectVersion() {
+  // We don't inject version in release since cache is reset there anyway
+  // We don't want to pollute tests with current version number so we disable it
+  // for all tests (except one)
+  if (isRelease() || process.env.REANIMATED_JEST_DISABLE_VERSION === 'jest') {
+    return false;
+  }
+  return true;
```

Comment:
```suggestion
  // We don't inject version in release since cache is reset there anyway
  if (isRelease()) {
    return false;
  }

  // We don't want to pollute tests with current version number so we disable it
  // for all tests (except one)
  if (process.env.REANIMATED_JEST_DISABLE_VERSION === 'jest') {
    return false;
  }

  return true;
```